scripting/include/srccoop/blackmesa/entitypatch.inc

```diff
@@ -1120,3 +1120,15 @@ public MRESReturn Hook_Dissolve(int _this, DHookReturn hReturn, DHookParam hPara
 	}
 	return MRES_Ignored;
 }
+
+// Switches between using singleplayer and multiplayer weapon values.
+// This will get triggered by the command `params_reload_server`.
+public MRESReturn Hook_CParamsManager_InitInstances(int _this, Handle hReturn, Handle hParams)
+{
+	// TODO:
+	// Currently, this is getting only called during level init, but
+	// we also need to call this if the feature is turned on/off mid-round.
+	CParamsManager pParamsManager = CParamsManager(view_as<Address>(_this));
```

Comment:
If the callback is defined with Address type, no need to cast

Reply:
Fixed.